apps/studio/schemas/organizer.ts

```diff
@@ -0,0 +1,34 @@
+import { HeartIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+export default defineType({
+  name: "organizer",
+  icon: HeartIcon,
+  title: "Organizers",
+  type: "document",
+  fields: [
+    defineField({
+      name: "name",
+      title: "Name",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "pronouns",
+      title: "Pronouns",
+      type: "string",
+    }),
+    defineField({
+      name: "linkedinURL",
+      title: "LinkedIn URL",
+      type: "url",
+      validation: (Rule) => Rule.required().uri({ scheme: ["http", "https"] }),
```

Comment:
`url` fields are restricted to http and https by default. Is there a reason for explicitly stating the accepted uris?

Reply:
Oops, I didn't know that it has this by default. No, there's no reason.

Reply:
Fixed in [`babb28d`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/babb28de063249dfcad0d491a4467125cb60e11d)